lighthouse-core/test/report/html/renderer/category-renderer-test.js

```diff
@@ -87,6 +87,17 @@ describe('CategoryRenderer', () => {
     assert.ok(auditDOM.matches('.lh-audit--informative'));
   });
 
+  it('renders audits with warnings', () => {
+    const auditResult = {
+      title: 'Audit',
+      description: 'Learn more',
+      warnings: ['It may not have worked!'],
+      score: 1,
+    };
+    const auditDOM = renderer.renderAudit({id: 'foo', score: 1, result: auditResult});
+    assert.ok(auditDOM.querySelector('.lh-debug--warnings'), 'did not render debug message');
```

Comment:
want to test at least that the warning text is in there as well? And probably want a plural/bulleted test in addition